spec/models/computacenter/api/cap_usage_update_spec.rb

```diff
@@ -0,0 +1,72 @@
+require 'rails_helper'
+
+RSpec.describe Computacenter::API::CapUsageUpdate do
+  let(:args) do
+    {
+      'capType' => 'DfE_RemainThresholdQty|Std_Device',
+      'shipTo' => '123456',
+      'capAmount' => 100,
+      'usedCap' => 20,
+    }
+  end
+  let(:update) { described_class.new(args) }
```

Comment:
```suggestion
  subject(:update) { described_class.new(args) }
```
is more idiomatic RSpec I believe...

Reply:
done, plus the consequent whack-a-mole Rubocop dance  :)